src/pypants/__init__.py

```diff
@@ -1,3 +1,3 @@
 """CLI for working with Python packages and BUILD files in a Pants monorepo"""
 
-__version__ = "0.5.1"
+__version__ = "0.6.0.dev1"
```

Comment:
No need to change while you're testing, but when this is ready to merge for real, let's change to 1.0.0. It's a major, breaking change.

Reply:
100% agreed. I'm not done making changes here and I figured this was fine for dev.

Reply:
this is now done